extensions/arc/package.json

```diff
@@ -780,13 +825,13 @@
                           "label": "%arc.postgres.server.group.memory.request%",
                           "variableName": "AZDATA_NB_VAR_POSTGRES_SERVER_GROUP_MEMORY_REQUEST",
                           "type": "number",
-                          "min": 0
+                          "min": 1
```

Comment:
For both the memory limit and request, let's make the min 256.  That matches the validation we have in azdata:

```
azdata arc postgres server create -n pg -ml 255Mi
Memory limit must be at least 256Mi

azdata arc postgres server create -n pg -mr 255Mi
Memory request must be at least 256Mi
```